samples/cicd/skaffold/README.md

```diff
@@ -0,0 +1,38 @@
+# Skaffold
+
+skaffold is a tool that enables fast development iteration and controls deployment to local or remote clusters
+
+If running `skaffold run` for deployment, manifests are pulled from remote charts, if running `skaffold dev` for development and hot reload, manifests are pulled from current branch.
```

Comment:
That is neat

Reply:
what is the requirement to have this at root? It intergrates well with cloudcode